packages/synapse-react-client/test/containers/SynapseTable/SynapseTableCell.integration.test.tsx

```diff
@@ -0,0 +1,348 @@
+import { render, screen } from '@testing-library/react'
```

Comment:
This were previously in the SynapseTable test suite, just pulled them out to their own file. No test changes.